docs/src/index.md

```diff
@@ -496,26 +474,33 @@ x_thread = @chain df begin
 end
 ```
 
-Another alternative is Pipe.jl which exports the `@pipe` macro for piping. 
-The piping mechanism in Pipe requires explicit specification of the piped
-object via `_` instead of assuming it is the first argument to the next function.
-The Pipe.jl equivalent of the above is:
+By default, `@chain` places the value of the 
+previous expression into the first argument of the current
+expression. The placeholder `_` is used to break that convention
+and refer to the argument returned from the previous 
+expression.
 
 ```julia
-using Pipe, Statistics
+@chain df begin 
+    map(_.a, _.x) do a, x
+        a + x
+    end
+end
```

Comment:
This example isn't super useful in practice. Maybe show a more realistic one? E.g. one where `_` is used after a few transformations?

Reply:
Added a better one.